### PR TITLE
Add endpoint to get manufacturer info.

### PR DESCRIPTION
**Pull request by doug-lovett**
Signed-off-by: Doug Lovett <doug@diamante.ca>

*Issue #:* /bcgov/entity#12642

*Description of changes:*
- Add new endpoint to get manufacturer information by account id (including submitting party).
- 12928 MHR staff search fee charges use CSRCS, MSRCS codes. For priority charge an additional PRIMH code.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: mhr_api/src/mhr_api/models/account_bcol_id.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+All crown charge accounts must have at least one record.
+"""
+
+from .db import db
+
+
+class AccountBcolId(db.Model):
+    """Map a user account ID to one or more BCOL account numbers."""
+
+    __versioned__ = {}
```
Comment (thorwolpert) [inline mhr_api/src/mhr_api/models/account_bcol_id.py]: this wont get used, as continuum isn't loaded and we don't want to use that.

File: mhr_api/src/mhr_api/models/account_bcol_id.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+"""This table manages a the mapping of BCRS account ID's to BCOL numbers.
+
+Link an account id to a legacy BCOL number to associate the account with a client party code.
+All crown charge accounts must have at least one record.
+"""
+
+from .db import db
+
+
+class AccountBcolId(db.Model):
```
Comment (thorwolpert) [inline mhr_api/src/mhr_api/models/account_bcol_id.py:23]: something to consider in the design, is that bcol will be gone in a year.

**Review by thorwolpert — COMMENTED** [review]
just a comment on the _versioned_ property

**Review by dimak1 — APPROVED** [review]
Looks good, few comments from Thor.

File: mhr_api/src/mhr_api/models/account_bcol_id.py (hunk 1 above)
Comment (doug-lovett) [reply mhr_api/src/mhr_api/models/account_bcol_id.py]: Removed in latest commit.

File: mhr_api/src/mhr_api/models/account_bcol_id.py (hunk 2 above)
Comment (doug-lovett) [reply mhr_api/src/mhr_api/models/account_bcol_id.py:23]: Noted: this table already exists in PPR to link registries accounts to bcol. Both PPR and MHR still use BCOL account numbers as identifiers. A general cleanup will be needed when BCOL goes away.

**Review by ChasBoucher — COMMENTED** [review]
LGTM

**Pull request merged**